docs/source/kedro_project_setup/configuration.md

```diff
@@ -208,7 +208,7 @@ except MissingConfigException:
 The `kedro.framework.context.KedroContext` class uses the approach above to load project parameters.
 ```
 
-Parameters can then be used on their own or fed in as function inputs, as described [below](#use-parameters).
+Parameters can then be used on their own or fed in as function inputs, as described [below in Use Parameters](#use-parameters).
```

Comment:
~Parameters can then be used on their own or fed in as function inputs, as described below in the ["Use parameters" section](#use-parameters).~

Reply:
Better yet IMO, and avoids weird quoting:
```suggestion
[Parameters can then be used on their own or fed in as function inputs.](#use-parameters)
```